modules/nf-core/bcftools/merge/main.nf

```diff
@@ -27,16 +27,17 @@ process BCFTOOLS_MERGE {
     def regions = bed ? "--regions-file $bed" : ""
     def extension = args.contains("--output-type b") || args.contains("-Ob") ? "bcf.gz" :
                     args.contains("--output-type u") || args.contains("-Ou") ? "bcf" :
+                    args.contains("--output-type z") || args.contains("-Oz") ? "vcf.gz" :
                     args.contains("--output-type v") || args.contains("-Ov") ? "vcf" :
                     "vcf.gz"
```

Comment:
```suggestion
                    "vcf"
```

The default is uncompressed VCF :)

Reply:
Right, of course..Thanks! Leaves me with two questions though:

- Should bcftools convert look the same?
- Why does bcftools norm have `def args = task.ext.args ?: '--output-type z'`? To not break stuff?

Reply:
Yeah there the bcftools modules are a bit of a wild wild west :grin: It doesn't really matter for now I think. You can update the other modules if you want or just leave them be, but I think this is the ideal setup that's the most flexible in every case. The reason `--output-type z` has been set as default in bcftools norm is because it's usually desired to create the bgzipped VCF by default. That's something you can do here too if you want that :)

Reply:
Thanks for the clarification. I think it's good like this, my main reason for the update was to allow other input files than *.vcf.gz and this solves it :)

Reply:
Perfect! Thank you for the contribution!